autoPyTorch/optimizer/smbo.py

```diff
@@ -53,6 +56,8 @@ def get_smac_object(
         ta_kwargs (typing.Dict[str, typing.Any]): Arguments to the above ta
         n_jobs (int): Amount of cores to use for this task
         dask_client (dask.distributed.Client): User provided scheduler
+        initial_configurations (typing.List[Configuration]): List of initial
+            configurations which smac will run before starting the search process
```

Comment:
[These variables](https://automl.github.io/SMAC3/master/apidoc/smac.facade.smac_ac_facade.html)?
If so, could you add the link?

Reply:
Could you explain your comment? I didn't get what you mean.